src/constants/customWalletsConfig.ts

```diff
@@ -4,7 +4,7 @@ const APP_NAME = 'PoolTogether'
 
 const INFURA_ID = process.env.NEXT_PUBLIC_INFURA_ID
 
-const RPC_URL =
+export const RPC_URL =
```

Comment:
this feels like it might be incorrect--any alternatives come to mind?

the widget uses an rpc url for read-only calls until the user has connected their wallet and we can pass it a `provider`

Reply:
There is a new function called `getRpcUrl` in the `@pooltogether/utilities` library. I've upgraded the `v4-ui` package to use the latest version, so we can pull the RPC URL from there.